core/api.go

```diff
@@ -444,7 +448,7 @@ func invalidlimit(limit string) bool {
 }
 
 func (core *Core) mustBeUnlocked(w http.ResponseWriter) bool {
-	status, err := core.vault.status()
+	status, err := core.vault.Status()
```

Comment:
... curses, I fixed this in my branch already.  I'll deal with the merge conflicts..